.github/workflows/test.yaml

```diff
@@ -18,7 +18,5 @@ jobs:
       channel: 1.28-strict/stable
       charmcraft-channel: latest/edge
       juju-channel: 3.4/stable
-      self-hosted-runner: true
-      self-hosted-runner-label: "edge"
```

Comment:
Why are you removing this?

Reply:
The integration tests failed to run in the self-hosted runner environment, and I can't reproduce the error locally. I will investigate this problem later, but for now, just switch to using the GitHub-hosted runners to merge this pull request.

Reply:
Ok, there were some infrastructure issues. I think the issues could be resolved. Please re-enable in the next PR.